src/utils/utils.ts

```diff
@@ -31,3 +33,26 @@ export async function executeCommand(command: string, cwd?: string): Promise<str
 export function generateQuotedFullName(schema: string, objectName: string): string {
 	return `[${escapeClosingBrackets(schema)}].[${escapeClosingBrackets(objectName)}]`;
 }
+
+
+/**
+ * Gets a unique file name
+ * Increment the file name by adding 1 to function name if the file already exists
+ * @param folderPath selected azure function project folder path
+ * @param fileName objectName that was chosen by the user
+ * @returns the unique file name that will always add at least `1` to the function name
```

Comment:
or undefined (and specify what case returns undefined)

Reply:
The returns comment needs to be updated too - it no longer always add a string to the end (same with the overall description)

Reply:
the word "promise" should probably exist here too